server/rotation.go

```diff
@@ -177,3 +177,73 @@ func (k keyRotater) rotate() error {
 	k.logger.Infof("keys rotated, next rotation: %s", nextRotation)
 	return nil
 }
+
+type RefreshTokenPolicy struct {
+	rotateRefreshTokens bool // enable rotation
+
+	absoluteLifetime  time.Duration // interval from token creation to the end of its life
+	validIfNotUsedFor time.Duration // interval from last token update to the end of its life
+	reuseInterval     time.Duration // interval within which old refresh token is allowed to be reused
+
+	Clock func() time.Time
```

Comment:
does it need to be exported? also I see that in other places such function is called just 'now' (see server.Server or server.Config)

Reply:
Yeah, fixed.